src/gui/touchscreengui.cpp

```diff
@@ -761,8 +762,13 @@ void TouchScreenGUI::handleReleaseEvent(int evt_id)
 			SEvent *translated = new SEvent;
 			memset(translated, 0, sizeof(SEvent));
 			translated->EventType               = EET_MOUSE_INPUT_EVENT;
-			translated->MouseInput.X            = m_move_downlocation.X;
-			translated->MouseInput.Y            = m_move_downlocation.Y;
+			if (m_use_crosshair || m_camera_mode == CAMERA_MODE_THIRD) {
+				translated->MouseInput.X            = m_screensize.X / 2;
+				translated->MouseInput.Y            = m_screensize.Y / 2;
+			} else {
+				translated->MouseInput.X            = m_move_downlocation.X;
+				translated->MouseInput.Y            = m_move_downlocation.Y;
```

Comment:
Incorrect alignment. Also, now that these lines are inside if-else blocks, the `=` needn't be aligned with the lines outside the blocks.

Reply:
Done. I also moved it to bottom.